ChangeLog.md

```diff
@@ -4,21 +4,176 @@ Changelog
 Notable changes to the FreeOrion project will be documented in this file.
 
 
-[v0.4.9+] - TBA
+[v0.4.10] - TBA
 ---------------
 
+### Key Changes
+
+- Python 3.5+ is now supported and required. Python 2 dependence is removed
+- Added a new ship weapon part, the Arc Disruptor, which is useful for countering many cheap "chaff" ships
+- Reworked weapon and fighter combat targeting to limit what each can target
```

Comment:
Fighter stats are completely rebalanced and the Heavy Bomber fighter type was added